### DISCUSSION TITLE
Usage guidelines

### DISCUSSION THREAD

**Discussion by tom-ricci** [cat-milk/Anime-Girls-Holding-Programming-Books#400] [Q&A] [answered]
I kind of want to make a site which loads a random picture from the repo on page load, but I'm not sure if this would be okay considering there's no license or attribution information. Can I do this, and if so, how (if at all) should I credit the contributors?

**Comment by asrvd** [comment, marked as answer]
the project is not licensed but since the images are work of contributors, you can just add a credit which links to this repo and you'll be good to go ig.
